web/src/main/java/org/akaza/openclinica/control/submit/ListNotesTableToolbar.java

```diff
@@ -136,7 +135,7 @@ public String enabled() {
          *      java.util.Locale)
          */
         String getIndexes() {
-            String result = "4, 5, 9, 11, 14, 16, 17, 19";
+            String result = "1, 4, 8, 10, 12, 16";

```

Comment:
Also need clarification here as to why this was changed.

Reply:
See comments above method.